utils/constants.ts

```
@@ -1,20 +1,9 @@
-  solution: '/solution',
-  sector: '/sectors',
-  engagementStage: '/engagement-stage',
-  currency: '/currencies',
-  leadDocuments: '/lead-documents',
-  roles: '/roles',
-  estimatedBudget: '/estimated-budgets', // Dedicated estimated budgets endpoint
-  currencies: '/currencies',
-} as const;
+// CRM API Configuration
+export const CRM_URL = process.env.NEXT_PUBLIC_CRM_URL || process.env.CRM_URL;
+export const CRM_API_URL = CRM_URL; // Alias for backward compatibility
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)